art/defences/preprocessor/spatial_smoothing_pytorch.py

```diff
@@ -138,15 +83,70 @@ def __init__(
         self.clip_values = clip_values
         self._check_params()
 
-        self.median_blur = self.MedianBlurCustom(kernel_size=(self.window_size, self.window_size))
-
         # Set device
         if device_type == "cpu" or not torch.cuda.is_available():
             self._device = torch.device("cpu")
         else:
             cuda_idx = torch.cuda.current_device()
             self._device = torch.device("cuda:{}".format(cuda_idx))
 
+        from kornia.filters import MedianBlur
+
+        class MedianBlurCustom(MedianBlur):
+            """
+            An ongoing effort to reproduce the median blur function in SciPy.
+            """
+
+            import torch  # lgtm [py/repeated-import]
```

Comment:
```suggestion
```
I think this line is not necessary anymore because there are another import of torch in the outside class.